client/pages/sections/establishments/index.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import { v4 } from 'uuid';
```

Comment:
You should import from `uuid/v4` to avoid including all the code for the other versions.

Reply:
good shout